tutorials/source-modeling/plot_fix_bem_in_blender.py

```diff
@@ -129,9 +138,68 @@
             op.join(bem_dir, 'inner_skull_orig.surf'))
 
 # Overwrite the original surface with the fixed version
+# In real study you should provide the correct metadata using ``volume_info=``
 mne.write_surface(op.join(bem_dir, 'inner_skull.surf'), coords, faces,
                   overwrite=True)
 
+###############################################################################
+# Editing the head surfaces
+# ^^^^^^^^^^^^^^^^^^^^^^^^^
+#
+# Sometimes the head surfaces are faulty and require manual editing. We use
+# :func:`mne.write_head_bem` to convert the fixed surfaces to ``.fif`` files.
+#
+# Low-resolution head
+# ~~~~~~~~~~~~~~~~~~~
+#
+# For EEG forward modeling, it is possible that ``outer_skin.surf`` would be
+# manually edited. In that case, remember to save the fixed version of
+# ``-head.fif`` from the edited surface file for coregistration.
+
+# Load the fixed surface
+coords, faces = mne.read_surface(op.join(bem_dir, 'outer_skin.surf'))
+
+# Make sure we are in the correct directory
+if 'flash' in bem_dir:
+    bem_dir = op.join(subjects_dir, 'sample', 'bem')
+
+# Remember to backup the original head file in advance!
+# Overwrite the original head file
+mne.write_head_bem(op.join(bem_dir, 'sample-head.fif'),
+                   coords, faces, overwrite=True)
+
+###############################################################################
+# High-resolution head
+# ~~~~~~~~~~~~~~~~~~~~
+#
+# We use :func:`mne.read_bem_surfaces` to read the head surface files. After
+# editing, we again output the head file with :func:`mne.write_head_bem`.
+# Here we use ``-head.fif`` for speed.
+
+# If ``-head-dense.fif`` does not exist, you need to run
+# ``mne make_scalp_surfaces`` first.
+# [0] because a list of surfaces is returned
+surf = mne.read_bem_surfaces(
+    op.join(bem_dir, 'sample-head.fif'))[0]
+
+# For consistency only
+coords = surf['rr']
+faces = surf['tris']
+
+# Write the head as an .obj file for editing
+mne.write_surface(op.join(conv_dir, 'sample-head.obj'),
+                  coords, faces, overwrite=True)
+
+# We use the same surface to simulate the fixed version here
+# Read in the .obj file
+coords, faces = mne.read_surface(
+    op.join(conv_dir, 'sample-head.obj'))
+
+# Remember to backup the original head file in advance!
+# Overwrite the original head file
+mne.write_head_bem(op.join(bem_dir, 'sample-head.fif'),
+                   coords, faces, overwrite=True)
```

Comment:
I would not actually overwrite these files, I'd leave these in a comment like
```suggestion
# This could be accomplished for example with::
#
#     >>> mne.write_head_bem(op.join(bem_dir, 'sample-head.fif'),
#                            coords, faces, overwrite=True)
```